rfcs/progress-bar-component.md

```diff
@@ -0,0 +1,67 @@
+# Progress Bar Component
+
+Progress bar is used to indicate a progress towards the completion of a task.
+
+## Usage
+
+### Basic usage
+
+```javascript
+import * as React from 'react';
+import {ProgressBar} from 'baseui/progress-bar';
+
+export default () => <ProgressBar value={55} />;
+```
+
+### Advanced usage
+
+```javascript
+import * as React from 'react';
+import {ProgressBar} from 'baseui/progress-bar';
+
+export default () => {
+  return
+    <ProgressBar
+      value={value}
+      getProgressInfo={(value) => `${value}% Loaded`}
+      overrides={{
+        Label: props => <CustomLabel>{props.children}</CustomLabel>,
+      }}
+    />;
+}
+```
+
+## Exports
+
+* `ProgressBar`
+* `StyledRoot`
+* `StyledBar`
+* `StyledLabel`
+
+## `ProgressBar` API
+
+* `getProgressInfo: (value) => node` - Optional
```

Comment:
`getProgressLabel` may work better here since I called the styled component `Label`